admin/templates/applications/application.html

```diff
@@ -27,6 +27,9 @@ <h4 class="govuk-heading-m">Marriage or civil partnership details</h4>
     <h4 class="govuk-heading-m">Payment details</h4>
     <ul class="govuk-list govuk-list--bullet">
         <li>Payment method: <strong>{{ application.submitAndPay.method }}</strong></li>
+        {% if application.submitAndPay.method == 'Online' %}
+            <li>Reference: <strong>{{ payment_details.reference }}</strong></li>
+        {% endif %}
```

Comment:
Similarly, we could do without the `payment_details`, since it's the same as our own reference number